driver/aws/rds.py

```diff
@@ -95,3 +95,33 @@ def get_db_type(db_instance_identifier: str, client: RDSClient) -> str:
     if db_type == "aurora":  # for aurora mysql 5.6
         db_type = "aurora_mysql"
     return db_type
+
+
+def get_db_parameter_group_name(db_instance_identifier: str, client: RDSClient) -> str:
+    """
+    Get database parameter group name
+    """
+    instance_info = get_db_instance_info(db_instance_identifier, client)
+    return instance_info["DBParameterGroups"][0]["DBParameterGroupName"]
```

Comment:
db instances can have multiple parameter groups? interesting

Reply:
Yeah I too would think only one should be allowed, as we are not allowed to specify more than one while [creating](https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html) or [modifying](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html) an instance:



<img width="1006" alt="image" src="https://user-images.githubusercontent.com/5196925/183722606-eb70a63d-f0a4-4970-9fb1-6c1d9302b203.png">


<img width="872" alt="image" src="https://user-images.githubusercontent.com/5196925/183722088-53948e3d-3c63-47ee-8b00-c0b459d9e590.png">

Reply:
You might want to use `.get()` and list length test to avoid the program aborting from `KeyError`